core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -150,14 +150,6 @@
             </li>
           </ul>
         </li>
-        <li class="nav-item" ng-mouseover="$ctrl.blurNavigationLinks($event)" ng-show="$ctrl.navElementsVisibilityStatus.I18N_TOPNAV_DONATE">
```

Comment:
Is this expected? The donate wouldn't be accessible then right?

Reply:
Ah so we show this in a dropdown under "Get involved". So it is OK to get rid of the additional button.